packages/flame_tiled/lib/src/tile_atlas.dart

```diff
@@ -95,9 +95,8 @@ class TiledAtlas {
       // The map contains one image, so its either an atlas already, or a
       // really boring map.
       final tiledImage = imageList.first;
-      final image =
-          (await Flame.images.load(tiledImage.source!, key: key)).clone();
```

Comment:
You mean `Flame.images.add(key, image);` is not needed?

Reply:
Actually yes, there is no need for adding `atlas{$source}` for the image key.
This is for backward compatibility for someone who is using this as a cache key.
But it can be eliminated of course.